tests/functional/disaster-recovery/regional-dr/conftest.py

```
@@ -449,6 +450,70 @@ def teardown():
+                (e.g. None, 'aggressive', 'passive', 'force')
+        """
+
+        cephfs_pool_name = constants.RDR_CUSTOM_CEPHFS_POOL
+        cephfs_sc_name = constants.RDR_CUSTOM_CEPHFS_STORAGECLASS
+
+        for cluster in get_non_acm_cluster_config():
+            config.switch_ctx(cluster.MULTICLUSTER["multicluster_index"])
+            # Create or verify existing SC in all clusters
+            existing_sc_list = get_all_storageclass()
+            if cephfs_sc_name in existing_sc_list:
+                log.info(f"Storage class {cephfs_sc_name} already exists")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)